tokio-threadpool/src/lib.rs

```diff
@@ -129,7 +129,7 @@ pub struct Shutdown {
 /// thread_pool.shutdown().wait().unwrap();
 /// # }
 /// ```
-#[derive(Debug)]
+#[derive(Debug, Clone)]
```

Comment:
Same here, it would be better to leave `Clone` off.

Reply:
How do I keep threadpool builder in this case? There would be no possibility to write oneliner like:

```
+        Builder::new()
+            .threadpool_builder(
+                threadpool::Builder::new()
+                    .name_prefix("tokio-runtime-worker-")
```

only code like:
```
let threadpool_builder = ...;
let runtime = runtime::Builder(......).build();
```

Reply:
Done